rules/linux/lnx_install_root_certificate.yml

```diff
@@ -0,0 +1,25 @@
+title: Install Root Certificate
+id: 78a80655-a51e-4669-bc6b-e9d206a462ee
+description: Detects installed new certificate
+author: Ömer Günal, oscd.community
+date: 2020/10/05
+tags:
+    - attack.defense_evasion
+    - attack.t1553.004
+level: low
+logsource:
+    product: linux
+detection:
+    selection:
+        - CommandLine|contains|all:
+          - 'mv '
+          - '/usr/local/share/ca-certificates'
+          - 'update-ca-certificates'
```

Comment:
Hello @omergunal! 

The sigma will convert this into a search query, that will search for all 3 of the listed patterns in **one** event. 
The problem here is that it will never happen. `update-ca-certificates` is a pattern for a command that will be executed after certificate will be updated (moved).

So you need to split `update-ca-certificates` into another section.
Also, please split commandline and processnames. It's cheaper from resources perspective and will be FP free.

Please apply this recommendation to the rest of your rules. Thank you very much!

Reply:
Thanks!